pkg/cmd/create/create_cluster_eks.go

```diff
@@ -124,6 +124,8 @@ func (o *CreateClusterEKSOptions) Run() error {
 		os.Exit(-1)
 	}
 
+	// We don't want to pass any args that were introduced by mistake because they are passed to all other commands down the chain
+	o.Args = []string{}
```

Comment:
I would also log a warning message before reseting the Args just to make the user aware of this case.

Reply:
Check it again, I added a warning log message.